src/commands/utils/uploadToAzure.ts

```diff
@@ -0,0 +1,64 @@
+import { logError } from "../../errors";
```

Comment:
This file might get renamed uploadToProvider at some point. Or we can throw the method in utils/uploadUDF